examples/with-storybook-styled-jsx-scss/.storybook/.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "next/babel",
+      {
+        "styled-jsx": {
+          "plugins": ["styled-jsx-plugin-sass"]
+        }
+      }
+    ]
+  ]
+}
```

Comment:
Can we remove this file? our current storybook example doesn't seem to need one 🤔

Reply:
this is necessary to be able to style the actual components inside storybook with styled-jsx and sass

Reply:
otherwise they won't style...

Reply:
I mean, this is the same `.babelrc` in the root folder, do we need it twice?

Reply:
Yea, I don't yet know how to get Storybook to use the default .babelrc file, perhaps there is a way

Reply:
I removed the file and it worked, can you try it again by removing the file and see if it works for you too? 🙏

Reply:
I've removed this file as everything seems to be working fine without it 🤔

Reply:
this is odd, but preferable!!! I have checked and it works...